log/logsink.go

```diff
@@ -214,11 +148,11 @@ func (l *Logger) log(msg string, context map[string]interface{}) {
 // the log line.  The key/value pairs can then be used to add additional
 // variable information.  The key/value pairs should alternate string
 // keys and arbitrary values.
-func (l *Logger) Info(msg string, keysAndValues ...interface{}) {
-	if !l.Enabled() {
+func (ls *Sink) Info(level int, msg string, keysAndValues ...interface{}) {
```

Comment:
nit. Usually function receivers use the first letter of the struct in small caps `(s *Sink)`